lmsweb/models.py

```diff
@@ -174,6 +172,25 @@ def course_get_administrators(course_id):
     return course.get_course_administrators()
 
 
+ALL_MODELS = (User, Course, Lecture)
+
+from flask_admin import Admin
+from flask_admin.contrib.peewee import ModelView
+
+# set optional bootswatch theme
+app.config['FLASK_ADMIN_SWATCH'] = 'cerulean'
+
+admin = Admin(app, name='LMS', template_mode='bootstrap3')
```

Comment:
template=bootstrap3?

Reply:
Since bootstrap 2 is the default we would like to use 3